Nodejs/Product/Nodejs/Project/NodeModulesNode.cs

```diff
@@ -359,6 +346,18 @@ private class NpmPathProvider : INpmPathProvider {
                 null, 1000, Timeout.Infinite);
         }
 
+        private static string GetStatusBarMessage(NpmCommandCompletedEventArgs e) {
+            if (e.WithErrors) {
+                return SR.GetString(
+                    e.Cancelled ? SR.NpmCancelledWithErrors : SR.NpmCompletedWithErrors,
+                    e.CommandText);
+            }
+            if (e.Cancelled) {
+                return SR.GetString(SR.NpmCancelled, e.CommandText);
+            }
+            return SR.GetString(SR.NpmSuccessfullyCompleted, e.CommandText);

```

Comment:
nit: even though it's fine to place the return within the if blocks like this, the code is more readable when you use if -> else if because it's clearer at a glance that the if blocks are mutually exclusive.